tests/bin/eslint.js

```diff
@@ -123,7 +123,7 @@ describe("bin/eslint.js", () => {
             const exitCodePromise = assertExitCode(child, 0);
             const stdoutPromise = getOutput(child).then(output => {
                 assert.strictEqual(output.stdout.trim(), expectedOutput);
-                assert.strictEqual(output.stderr, "");
+                assert.match(output.stderr, /ESLintIgnoreWarning: The ".eslintignore" file is no longer supported./u);
```

Comment:
Can we prevent this test from having this warning? Would be best that only the dedicated test for this scenario exhibits this warning.

Reply:
Yes, if we remove the root `.eslintignore` this should go away.